client/lib/analytics/utils/split-wpcom-jetpack-cart-info.ts

```diff
@@ -1,40 +1,54 @@
-import { isJetpackPlan, isJetpackProduct } from '@automattic/calypso-products';
+import { isJetpackPlan, isJetpackProduct, isAkismetProduct } from '@automattic/calypso-products';
 import { ResponseCart, ResponseCartProduct } from '@automattic/shopping-cart';
 import costToUSD from './cost-to-usd';
 
 export type WpcomJetpackCartInfo = {
-	wpcomProducts: ResponseCartProduct[];
+	akismetProducts: ResponseCartProduct[];
 	jetpackProducts: ResponseCartProduct[];
-	containsWpcomProducts: boolean;
+	wpcomProducts: ResponseCartProduct[];
+	containsAkismetProducts: boolean;
 	containsJetpackProducts: boolean;
+	containsWpcomProducts: boolean;
+	akismetCost: number;
 	jetpackCost: number;
 	wpcomCost: number;
+	akismetCostUSD: number;
 	jetpackCostUSD: number;
 	wpcomCostUSD: number;
 	totalCostUSD: number;
 };
```

Comment:
Order here may look weird, I just reordered to have the categories listed in this order for all types of data
1. Akismet
2. Jetpack
3. WPCOM